.ci/conformance/tools/elastic_net/download_test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+echo
+echo "Test elastic net regression download started."
+echo
+wget -O test_elastic_net.py https://raw.githubusercontent.com/scikit-learn/scikit-learn/0.21.3/sklearn/linear_model/tests/test_coordinate_descent.py
```

Comment:
This is a lot of  duplication for this actions in multiple separate scripts. will be hard to made changes and align here.